components/builder-web/index.html

```diff
@@ -18,8 +24,18 @@
     <link id="hab-css" rel="stylesheet" href="/assets/app.css">
     <link rel="search" href="/opensearch.xml" type="application/opensearchdescription+xml" title="habitat.sh">
     <base href="/">
+
+    <!-- Heap -->
```

Comment:
@cnunciato I think the reason we're seeing the data appear in two places (in Heap) is that we're sending it directly to Heap with this snippet, and then again to Heap via Segment using the snippet below this. That would explain why we're seeing the company.id and company.name on the Production project in Heap, since where sending those two bits of data using a segment.identity call where Segment is using our Production Heap project ID.

Perhaps the solution here would be to use a different Segment snippet (Source) that points to the Development project. Let's look at Segment together when you have a few minutes.

Reply:
I did some poking around on Segment... I think what we need to do is create a new Source on Segment (localhost/local habitat dev) and add a new Heap destination to that Source (that uses the Heap 'Development' environment project ID).

Then we can send the identity info straight to heap using these Segment calls (that translate to the Heap `identity` and `track` calls --> https://segment.com/docs/destinations/heap/